frontend/src/components/ItemsList/Station.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import FuelListing from "./FuelListing";
+import "../../../style/itemsList.css";
+import StationInfo from "./StationsInfo";
+
+function Station() {
+  return (
+    <div className="station">
+      <StationInfo />
+      <div className="eltListing">
```

Comment:
This div is surrounded your component FuelListing, wich is already a div with items inside. You might not need these two div, pick one and delete the other.